src/uct/ib/ud/verbs/ud_verbs.c

```diff
@@ -393,6 +393,20 @@ uct_ud_verbs_iface_poll_tx(uct_ud_verbs_iface_t *iface, int is_async)
     return 1;
 }
 
+static UCS_F_ALWAYS_INLINE size_t uct_ud_iface_get_gid_len(void *packet)
```

Comment:
uct_ud_verbs_iface_get_gid_len

Reply:
Fixed in the next push. Coming once we have resolved one non-trivial comment.